Correct JES references and other mods to install list (#1285)
keptn/keptn.github.io: content/docs/install/istio/index.md

---

> Istio can be installed in order to access the services that Keptn deploys.

Reviewer: ```suggestion
Istio needs to installed in order to access the services that Keptn deploys using `helm-service`.
```
Though this is only true IF you are using `helm-service`, and applies to the cluster where `helm-service` is installed (this matters in a multi-cluster setup)

Author: done

Author: done

- /docs/1.0.x/continuous_delivery/expose_services/
---

Istio must be installed in order to access the services that Keptn deploys using `helm-service`.

> You can instead use and install the [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)

Reviewer 2: ```suggestion
If you don't want to use Istio or have no need for it, you can instead install and use the [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
```
